services/app-api/src/emailer/emails/sendQuestionCMSEmail.ts

```diff
@@ -0,0 +1,77 @@
+import { packageName as generatePackageName } from '../../../../app-web/src/common-code/healthPlanFormDataType'
+import { formatCalendarDate } from '../../../../app-web/src/common-code/dateHelpers'
+import { pruneDuplicateEmails } from '../formatters'
+import type { EmailConfiguration, EmailData, StateAnalystsEmails } from '..'
+import type { ProgramType, Question } from '../../domain-models'
+import {
+    stripHTMLFromTemplate,
+    renderTemplate,
+    findContractPrograms,
+} from '../templateHelpers'
+import { submissionQuestionResponseURL } from '../generateURLs'
+import type { ContractRevisionWithRatesType } from '../../domain-models/contractAndRates'
+
+export const sendQuestionCMSEmail = async (
+    contractRev: ContractRevisionWithRatesType,
+    stateAnalystsEmails: StateAnalystsEmails,
+    config: EmailConfiguration,
+    statePrograms: ProgramType[],
+    question: Question
+): Promise<EmailData | Error> => {
+    let receiverEmails = [...stateAnalystsEmails, ...config.devReviewTeamEmails]
+    if (question.addedBy.divisionAssignment === 'DMCP') {
+        receiverEmails.push(...config.dmcpReviewEmails)
+    } else if (question.addedBy.divisionAssignment === 'OACT') {
+        receiverEmails.push(...config.oactEmails)
+    }
+    receiverEmails = pruneDuplicateEmails(receiverEmails)
+
+    //This checks to make sure all programs contained in submission exists for the state.
+    const packagePrograms = findContractPrograms(contractRev, statePrograms)
+    if (packagePrograms instanceof Error) {
+        return packagePrograms
+    }
+
+    const packageName = generatePackageName(
+        contractRev.contract.stateCode,
+        contractRev.contract.stateNumber,
+        contractRev.formData.programIDs,
+        packagePrograms
+    )
+
+    const questionResponseURL = submissionQuestionResponseURL(
+        contractRev.contract.id,
+        config.baseUrl
+    )
+    const roundNumber = question.responses.length + 1
```

Comment:
<img width="985" alt="Screenshot 2023-12-05 at 11 31 00 AM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/21cffc7d-cccb-478c-9991-ddf839f58afe">

Based on this UI, it appears that a "round" is based on the number of questions and not number of responses. It looks like `HealthPlanPackageType` has a `questions` field, should a contract have the same? cc @macrael @JasonLin0991

Reply:
Good catch; rounds are based on the number of questions, not the responses. 

As for the `questions` field, I think eventually the contract will have the same, but since the frontend is still using strictly `HealthPlanPackageType`, we won't need to add it yet. What do you think @macrael?

Reply:
Okay cool, and I see there's an existing postgres function to get all questions for a contract that I can use. So I agree with nothing needing to change yet